src/Compilers/Core/Portable/Operations/ISuppressNullableWarningExpression.cs

```diff
@@ -1,16 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-namespace Microsoft.CodeAnalysis.Operations
-{
-    /// <summary>
-    /// Represents an expression that suppresses nullable warnings.
-    /// </summary>
-    /// <remarks>
-    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
-    /// change it in the future.
-    /// </remarks>
-    public interface ISuppressNullableWarningOperation : IOperation
```

Comment:
Not sure I agree with this part of the change: I think we still want an explicit suppression node in the tree, as we currently don't have another way of conveying that information (and I don't like the idea of introducing a flag on every IOperation node for suppression.)

Reply:
This is not the final design. In discussion of bound tree representation, we touched on this, but did not conclude whether suppression should be there, should be represented as a node, or should be represented as a flag.
Follow-up is https://github.com/dotnet/roslyn/issues/32881